runtime/printexc.c

```diff
@@ -142,12 +142,6 @@ void caml_fatal_uncaught_exception(value exn)
   handle_uncaught_exception =
     caml_named_value("Printexc.handle_uncaught_exception");
 
-  /* If the callback allocates, memprof could be called. In this case,
-     memprof's callback could raise an exception while
-     [handle_uncaught_exception] is running, so that the printing of
-     the exception fails. */
-  caml_memprof_set_suspended(1);
-
```

Comment:
Was this code removed on purpose of just as a merging accident?

Reply:
This was intentional: we removed all `caml_memprof_XXX` calls (outside `memprof.{c,h}`) from the runtime until statmemprof is working.
We didn't want some `caml_memprof_XXX` calls in place (e.g. `printexc.c`, `st_stubs.c`, `intern.c`, etc) but others (e.g. major and minor GC) missing.